src/runtime/graph/graph_runtime.cc

```diff
@@ -398,14 +398,8 @@ void GraphRuntime::SetupOpExecs() {
         }
       }
     } else if (inode.op_type == "_tensorrt_subgraph_op") {
-#ifdef TVM_GRAPH_RUNTIME_TENSORRT
-      CHECK_EQ(inode.subgraphs.size(), 1U) << "Only supports one subgraph per node";
-      CHECK_EQ(inode.subgraphs[0].arg_nodes.size(), inode.inputs.size());
-      op_execs_[nid] = tensorrt_exec_manager_.CreateExec(
-          inode.name, inode.subgraphs[0], args);
-#else
-      LOG(FATAL) << "TensorRT NOT enabled for operator " << inode.op_type;
-#endif  // TVM_GRAPH_RUNTIME_TENSORRT
+        LOG(FATAL) << "_tensorrt_subgraph_op is deprecated" << std::endl
```

Comment:
This change is okay to me. We can even remove this entire else if block because it was only used for NNVM TRT. The new Relay/TRT invocation doesn't need it. @trevor-m can you confirm that we don't use it in the downstream?